share/picongpu/tests/CurrentDeposition/bin/validate.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+#
+# This file is part of PIConGPU.
+# Copyright 2023 PIConGPU contributors
+# Authors: Hannes Wolf, Mika Soren Voss, Rene Widera
+# License: GPLv3+
+#
+help()
+{
+  echo "Validate CurrentDeposition output data."
+  echo "The test is evaluating the current density field with the corresponding analytic solution."
+  echo ""
+  echo "Usage:"
+  echo "    validate.sh [-d dataPath] [inputSetPath]"
+  echo ""
+  echo "  -d | --data dataPath                 - path to simulation output data"
+  echo "                                         Default: inputPath/simOutput/simData_%T.h5"
+  echo "  -h | --help                          - show help"
+  echo ""
+  echo "  inputSetPath                         - path to directory of python test files"
```

Comment:
The name of this option is misleading. It is not pointing to PIConGPU's input set, but rather to the directory w/ the analysis scripts. I think you can remove it.

Reply:
done